### PR TITLE
bump changelog

### PR DESCRIPTION
**Pull request by ngokevin**
**Changes proposed:**
- Replaced everything with dashes (sorry in advance for the diff mess).
- Added new changelog items at the bottom of each section.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,50 +8,64 @@
...
+- `aframe-core` merged with `aframe`. `window.AFRAME` exposes what `aframe-core` was exposing previously (e.g., `AFRAME.registerComponent` vs.  `AFRAME.aframeCore.registerComponent`). (#368)
+- `<a-assets>` must be declared within `<a-scene>`. (#910)
+- `<a-entity>.object3D` is now a `THREE.Group`. Use `<a-entity>.setObject3D` API to add new 3D objects from components. (#847)
+- Bumped three.js to r74 stable. (#1006)
+- npm points to a prebuilt bundle of `dist/aframe.js`.
+- Scene `canvas` elements are now appended to the scene by default rather than to the document body. The scene can specify the canvas to render to. (c0aa360)
+- Primitives such as `<a-sphere>` directly extend `<a-entity>` rather than template them. They can be registered with `AFRAME.registerPrimitive`. (#883)
+- `<a-template>` and HTML imports logic have been removed. Use https://github.com/ngokevin/aframe-template-component in the meantime. (#883)
+- `<a-camera>` no longer creates a cursor on its own. Do `<a-camera><a-cursor></a-cursor></a-camera>` instead. (#883)
+- Default geometry `depth`, `height`, `width` property values changed from 2 to 1. (#1245)
```
Comment (cvan) [inline CHANGELOG.md]: maybe

```
`2` to `1`
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,50 +8,64 @@
...
-* `<a-template>` and HTML imports logic have been removed. (#883)
-* `<a-camera>` no longer creates a cursor. Use `<a-cursor>` instead. (#883)
+- `aframe-core` merged with `aframe`. `window.AFRAME` exposes what `aframe-core` was exposing previously (e.g., `AFRAME.registerComponent` vs.  `AFRAME.aframeCore.registerComponent`). (#368)
+- `<a-assets>` must be declared within `<a-scene>`. (#910)
+- `<a-entity>.object3D` is now a `THREE.Group`. Use `<a-entity>.setObject3D` API to add new 3D objects from components. (#847)
+- Bumped three.js to r74 stable. (#1006)
+- npm points to a prebuilt bundle of `dist/aframe.js`.
+- Scene `canvas` elements are now appended to the scene by default rather than to the document body. The scene can specify the canvas to render to. (c0aa360)
+- Primitives such as `<a-sphere>` directly extend `<a-entity>` rather than template them. They can be registered with `AFRAME.registerPrimitive`. (#883)
+- `<a-template>` and HTML imports logic have been removed. Use https://github.com/ngokevin/aframe-template-component in the meantime. (#883)
```
Comment (cvan) [inline CHANGELOG.md]: nice :+1:

File: CHANGELOG.md [hunk 3]
```diff
@@ -8,50 +8,64 @@
...
-* Scene `canvas` elements are now appended to the scene by default rather than to the document body. The scene can specify the canvas to render to. (c0aa360)
-* Primitives such as `<a-sphere>` directly extend `<a-entity>` rather than template them. (#883)
-* `<a-template>` and HTML imports logic have been removed. (#883)
-* `<a-camera>` no longer creates a cursor. Use `<a-cursor>` instead. (#883)
+- `aframe-core` merged with `aframe`. `window.AFRAME` exposes what `aframe-core` was exposing previously (e.g., `AFRAME.registerComponent` vs.  `AFRAME.aframeCore.registerComponent`). (#368)
+- `<a-assets>` must be declared within `<a-scene>`. (#910)
+- `<a-entity>.object3D` is now a `THREE.Group`. Use `<a-entity>.setObject3D` API to add new 3D objects from components. (#847)
+- Bumped three.js to r74 stable. (#1006)
+- npm points to a prebuilt bundle of `dist/aframe.js`.
+- Scene `canvas` elements are now appended to the scene by default rather than to the document body. The scene can specify the canvas to render to. (c0aa360)
```
Comment (cvan) [inline CHANGELOG.md]: maybe say `<canvas>` to be consistent with usage elsewhere

File: CHANGELOG.md [hunk 4]
```diff
@@ -8,50 +8,64 @@
...
-* `<a-assets>` must be declared within `<a-scene>`. (#910)
-* `<a-entity>.object3D` is now a `THREE.Group`. Use `<a-entity>.setObject3D` API to add new 3D objects from components. (#847)
-* Bumped three.js to r74 stable. (#1006)
-* npm points to a prebuilt bundle of `dist/aframe.js`.
-* Scene `canvas` elements are now appended to the scene by default rather than to the document body. The scene can specify the canvas to render to. (c0aa360)
-* Primitives such as `<a-sphere>` directly extend `<a-entity>` rather than template them. (#883)
-* `<a-template>` and HTML imports logic have been removed. (#883)
-* `<a-camera>` no longer creates a cursor. Use `<a-cursor>` instead. (#883)
+- `aframe-core` merged with `aframe`. `window.AFRAME` exposes what `aframe-core` was exposing previously (e.g., `AFRAME.registerComponent` vs.  `AFRAME.aframeCore.registerComponent`). (#368)
+- `<a-assets>` must be declared within `<a-scene>`. (#910)
```
Comment (cvan) [inline CHANGELOG.md]: "must live in"? idk, "declared" sounds weird

File: CHANGELOG.md (hunk 2 above)
Comment (cvan) [inline CHANGELOG.md]: maybe capitalise `imports`?

File: CHANGELOG.md [hunk 5]
```diff
@@ -8,50 +8,64 @@
...
 
 ### Deprecations
 
-* `loader` component deprecated in favor of `collada-model` and `obj-model`. (#913)
-* `<a-model>` deprecated in favor of `<a-collada-model>` and `<a-obj-model>`. (#883)
-* `<a-EVENTNAME>` elements such as `<a-mouseenter>` deprecated in favor of `<a-event name="EVENTNAME">` (unstable). (#883)
-* `<a-cube>` deprecated in favor of `<a-box>`. (#883)
+- `loader` component deprecated in favor of `collada-model` and `obj-model`. (#913)
+- `<a-model>` deprecated in favor of `<a-collada-model>` and `<a-obj-model>`. (#883)
+- `<a-EVENTNAME>` elements such as `<a-mouseenter>` deprecated in favor of `<a-event name="EVENTNAME">` (unstable). (#883)
```
Comment (cvan) [inline CHANGELOG.md]: what's `(unstable)` mean?

File: CHANGELOG.md [hunk 6]
```diff
@@ -8,50 +8,64 @@
...
-* Support for loading *.OBJ* and .MTL assets. (#788)
-* *Texture caching* for better performance when reusing textures. (#1116)
-* Components can be attached to primitives (e.g., `<a-sphere>`). (#883)
-* Introduce *systems* API (unstable) to provide global scope and services for components. (#924)
-* Entities, including the scene, wait for their children to load before emitting the `loaded` event. (a8a4f06)
-* Entities emit `child-attach` when children are attached.
-* Lot of `<a-scene>` logic moved to configurable components. (#776)
-* Support for *multiple cameras* in a scene and switching between them. (#745)
-* Added more events for scene VR mode, material component, model components, sound component.
+- Introduce *shaders* to extend the material component and to register custom GLSL shaders. (#861)
```
Comment (cvan) [inline CHANGELOG.md]: should probably use past tense to be consistent with usage elsewhere:

`Introduce` -> `Introduced`

File: CHANGELOG.md [hunk 7]
```diff
@@ -8,50 +8,64 @@
...
+- Component *property types*. Property types define how a component property is parsed and stringified. Custom property types can be registered or defined inline with the property in the schema. Built-in property types include `array`, `boolean`, `color`, `int`, `number`, `selector`, `selectorAll`, `string`, `vec2`, `vec3`, `vec4`. (d35e56e)
+- *Single-property components*. A component can define itself as consisting of only one property by specifying a type and/or a default value in the schema. (d35e56e)
+- *Asset management system* that blocks scene render
```
Comment (cvan) [inline CHANGELOG.md]: perhaps more detail here? add a period too.

File: CHANGELOG.md [hunk 8]
```diff
@@ -8,50 +8,64 @@
...
+- *Play/pause* methods on entities and play/pause handlers on components. (9238861)
+- *Tick* method on components to register a function called on each scene tick. (#823)
+- Support for loading *.OBJ* and .MTL assets. (#788)
```
Comment (cvan) [inline CHANGELOG.md]: bold `.MTL` too. and perhaps backtick them both too.

File: CHANGELOG.md [hunk 9]
```diff
@@ -8,50 +8,64 @@
...
+- *Texture caching* for better performance when reusing textures. (#1116)
+- Components can be attached to primitives (e.g., `<a-sphere>`). (#883)
+- Introduce *systems* API (unstable) to provide global scope and services for components. They can be registered wtih `AFRAME.registerSystem`. (#924)
```
Comment (cvan) [inline CHANGELOG.md]: which parts are "unstable"? and should we say "experimental" instead?

File: CHANGELOG.md [hunk 10]
```diff
@@ -8,50 +8,64 @@
...
+- Entities, including the scene, wait for their children to load before emitting the `loaded` event. (a8a4f06)
+- Entities emit `child-attach` when children are attached.
+- Most `<a-scene>` logic moved to configurable components and systems. (#776)
+- Support for *multiple cameras* in a scene and switching between them. (#745)
+- Added more events for scene VR mode, material component, model components, sound component.
```
Comment (cvan) [inline CHANGELOG.md]: `and`?

Comment (cvan) [inline CHANGELOG.md]: do you want to list the events - or too many to list? maybe link to docs? but probably not necessary.

File: CHANGELOG.md [hunk 11]
```diff
@@ -8,50 +8,64 @@
...
+- Default geometry `segments*` property values increased for smoother meshes. (#1245)
+- Added more meta tags and automatically add properties to video elements to support inline video playback on iOS. (#316)
```
Comment (cvan) [inline CHANGELOG.md]: "Added meta tags for mobile web-app capabilities and automatically set attributes to video elements …"?

File: CHANGELOG.md [hunk 12]
```diff
@@ -8,50 +8,64 @@
...
+- Added three.js stats to the stats component. (#1223)
+- Added `<a-torus>` primitive. (#1184)
+- Texture caching. (#1116)
 
 ## Fixes
 
-* Stop `<a-animation>` when detached. (#727)
-* Fix `<a-animation>` begin attribute. (#885)
+- Stop `<a-animation>` when detached. (#727)
+- Fixed `<a-animation>` begin attribute. (#885)
```
Comment (cvan) [inline CHANGELOG.md]: backtick `begin`?

File: CHANGELOG.md [hunk 13]
```diff
@@ -8,50 +8,64 @@
...
+- Keyboard shortcuts no longer trigger when used alongside modifier keys. (#1211)
+- Fixed viewport issues in Twitter webview. (#1174)
```
Comment (cvan) [inline CHANGELOG.md]: `on iOS`?

File: CHANGELOG.md [hunk 14]
```diff
@@ -8,50 +8,64 @@
...
+- Raycaster and cursor can now intersect with loaded models. (#1166)
```
Comment (cvan) [inline CHANGELOG.md]: `components`?

**Comment by cvan** [comment]
great changes. thanks for doing this; I know it's a lot of work.

File: CHANGELOG.md (hunk 4 above)
Comment (ngokevin) [inline CHANGELOG.md]: probably fine, it's declarative markup. html elements don't need shelter.

File: CHANGELOG.md (hunk 5 above)
Comment (ngokevin) [inline CHANGELOG.md]: you know diego and i have some qualms about it, and it's unreviewed. i'm not certain what is going to happen later with this API so i just want to give a heads up about that.

Comment (cvan) [inline CHANGELOG.md]: yeah, that's fair then. unstable is scarier than experimental. so let's stick with that verbiage.

File: CHANGELOG.md (hunk 9 above)
Comment (ngokevin) [inline CHANGELOG.md]: because it's brand new

File: CHANGELOG.md (hunk 5 above)
Comment (ngokevin) [inline CHANGELOG.md]: experiemental actually fits better. but yeh either way.

**Pull request merged**